docs/list_of_tutorials.jl

```diff
@@ -20,6 +20,7 @@ if generate_tutorials
 
     tutorials = [
         "Atmos" => [
+            "Density current" => "Atmos/densitycurrent.jl",
```

Comment:
I think @jiahe23 would then just need to add this to the summary of tutorials page, or, if that PR has already gone through, could you add this to the summary page?

https://github.com/CliMA/ClimateMachine.jl/pull/1575